sycl/test/basic_tests/accessor/accessor_implicit_conversion_read.cpp

```diff
@@ -0,0 +1,22 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
```

Comment:
I suggest to refactor this into a unit-test: we don't really need to invoke device compilation for this example and can save some testing time on that. You can find some examples of accessor unit-tests [here](https://github.com/intel/llvm/tree/sycl/sycl/unittests/accessor)

Reply:
thanks @AlexeySachkov, done.